pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: test
-version: 0.12.21
+version: 0.12.21-dev
```

Comment:
0.12.22-dev. We want this version to be greater than  the last released version, so if people depend on it via git the versioning semantics are correct.